Dog.py

```diff
@@ -1,7 +1,10 @@
 from Canine import Canine
 
+
 class Dog(Canine):
-	def __init__(self, name):
+	def __init__(self, name, makeNoise):
 		super().__init__(name)
+		self.makeNoise = makeNoise
+
```

Comment:
Please use super class's constructor instead of using private attribute self.makeNoise in this class.

Like:
		super().__init__(name, makeNoise)